src/tactic/lean_core_docs.lean

```diff
@@ -0,0 +1,650 @@
+  decl_names := [`tactic.interactive.induction],
+  tags       := ["core", "basic", "induction"] }
+
+add_tactic_doc
+{ name       := "injection",
+  category   := doc_category.tactic,
+  decl_names := [`tactic.interactive.injection],
+  tags       := ["core", "structures", "induction"] }
+
+add_tactic_doc
+{ name       := "injections",
+  category   := doc_category.tactic,
+  decl_names := [`tactic.interactive.injections],
+  tags       := ["core", "structures", "induction"] }
+
+add_tactic_doc
+{ name       := "intro",
+  category   := doc_category.tactic,
+  decl_names := [`tactic.interactive.intro],
+  tags       := ["core", "basic", "logic"] }
+
+add_tactic_doc
+{ name       := "intros",
+  category   := doc_category.tactic,
+  decl_names := [`tactic.interactive.intros],
+  tags       := ["core", "basic", "logic"] }
+
+add_tactic_doc
+{ name       := "introv",
+  category   := doc_category.tactic,
+  decl_names := [`tactic.interactive.introv],
+  tags       := ["core", "logic"] }
+
+add_tactic_doc
+{ name       := "iterate",
+  category   := doc_category.tactic,
+  decl_names := [`tactic.interactive.iterate],
+  tags       := ["core", "combinator"] }
+
+add_tactic_doc
+{ name       := "left",
+  category   := doc_category.tactic,
+  decl_names := [`tactic.interactive.left],
+  tags       := ["core", "basic", "logic"] }
+
+/--
+`let h : t := p` adds the hypothesis `h : t := p` to the current goal if `p` a term of type `t`. If `t` is omitted, it will be inferred.
+
+`let h : t` adds the hypothesis `h : t := ?M` to the current goal and opens a new subgoal `?M : t`. The new subgoal becomes the main goal. If `t` is omitted, it will be replaced by a fresh metavariable.
+
+If `h` is omitted, the name `this` is used.
+
+Note the related mathlib tactic `set a := t with h`, which adds the hypothesis `h : a = t` to
+the local context and replaces `t` with `a` everywhere it can.
+-/
+add_tactic_doc
+{ name       := "let",
+  category   := doc_category.tactic,
+  decl_names := [`tactic.interactive.let],
+  tags       := ["core", "basic", "logic", "context management"] }
+
+add_tactic_doc
+{ name       := "mapply",
+  category   := doc_category.tactic,
+  decl_names := [`tactic.interactive.mapply],
+  tags       := ["core", "lemma application"] }
+
+add_tactic_doc
+{ name       := "match_target",
+  category   := doc_category.tactic,
+  decl_names := [`tactic.interactive.match_target],
+  tags       := ["core", "testing", "goal management"] }
+
+add_tactic_doc
+{ name       := "refine",
+  category   := doc_category.tactic,
+  decl_names := [`tactic.interactive.refine],
+  tags       := ["core", "basic", "lemma application"] }
+
+/--
+This tactic applies to a goal whose target has the form `t ~ u` where `~` is a reflexive relation,
+that is, a relation which has a reflexivity lemma tagged with the attribute `[refl]`.
+The tactic checks whether `t` and `u` are definitionally equal and then solves the goal.
+-/
+add_tactic_doc
+{ name       := "refl",
```

Comment:
```suggestion
{ name       := "refl / reflexivity",
```

Reply:
Currently we're leaving `reflexivity` completely undocumented. Is this what we want?

Reply:
It looks like it's included in the `decl_names` below, so I think this is fine.

Reply:
The `/` additions look good to me. You could also change the text in the doc string to mention both, e.g. "The tactic `refl` (or equivalently `reflexivity`) ..."